### PR TITLE
Update filehost.md

### PR DESCRIPTION
**Pull request by lf94**

### CODE REVIEW

File: extensions/filehost.md [hunk 1]
```diff
@@ -27,11 +27,11 @@ This specification offers a way for network administrators to recommend a hostin
 
 ## Architecture
 
-This specification introduces the `draft/FILEHOST` isupport token.
+This specification introduces the `draft/FILEHOST` RPL_ISUPPORT token. Please see https://modern.ircdocs.horse/#feature-advertisement for what an RPL_ISUPPORT token is.
```
Comment (progval) [inline extensions/filehost.md:30]: I don't think that is needed. Other specs don't link to it either, and googling "irc rpl_isupport" gives you good results.

File: extensions/filehost.md [hunk 2]
```diff
@@ -27,11 +27,11 @@ This specification offers a way for network administrators to recommend a hostin
...
 
-Its value MUST be a (possibly empty) space-separated list of URIs, and they SHOULD use either the `http` or `https` schemes. 
+Its value MUST be a (possibly empty) comma-separated list of URIs, and they MUST use the `https` scheme. This is due to there being privacy concerns while in private communication channels or channels which require TLS. Allowing `http` would enable an attacker to see what people are sharing in a private context.
```
Comment (progval) [inline extensions/filehost.md:32]: That's up to the implementations. There can be legitmate reasons to use HTTP.

File: extensions/filehost.md (hunk 1 above)
Comment (lf94) [reply extensions/filehost.md:30]: Yeah, true.

File: extensions/filehost.md (hunk 2 above)
Comment (lf94) [reply extensions/filehost.md:32]: Now that I think about it, yeah, if it's a concern of the implementation, they will just enforce https.

**Comment by lf94** [comment]
@progval could you just change the space-separated to comma-separated in your version? I'll close this PR.

**Comment by progval** [comment]
Why? Commas are valid in URLs while spaces aren't

**Comment by lf94** [comment]
> Why? Commas are valid in URLs while spaces aren't

Spaces separate ISUPPORT tokens

**Pull request closed without merging**